snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb

```diff
@@ -0,0 +1,44 @@
+Imports System  
+Imports System.IO  
+Imports System.Net  
+Imports System.Text  
```

Comment:
Here again, I don't think any System.Text types are used.

Reply:
According to VS, Encoding needs System.Text in Dim byteArray As Byte() = Encoding.UTF8.GetBytes(postData)